### PR TITLE
Let Lighthouse wait a bit longer for FCP

### LINKED ISSUE #105: Lighthouse error: LHError: NO_FCP
Some users reported an error with Lighthouse that seems to be a common issue when it takes too much time for a page to render on screen.

https://github.com/GoogleChrome/lighthouse/issues/7415

### PR DESCRIPTION
**Pull request by grgur**
Increased max FCP timeout in lighthouse to avoid known issues that occur when pages are taking a long time to paint. These projects are the ones that need good value from Gimbal 😄 

Fixes #105

### CODE REVIEW

**Comment by akagr** [comment]
@grgur While 60 seconds should be generous enough for most projects, is there value in exposing that as gimbal config?

**Comment by grgur** [comment]
Good thinking. I think we should have a common interface for forwarding any Lighthouse-specific config to Lighthouse

File: src/module/lighthouse/index.ts [hunk 1]
```diff
@@ -18,12 +18,21 @@ import parseReport from './output';
...
+  // Default config options
+  const defaults = {
+    output: ['json'],
+    maxWaitForFcp: 60 * 1000,
+  };
+
+  // Build options but let users change the defaults if needed
+  const options = {
+    ...defaults,
+    ...userOptions,
```
Comment (mitchellsimoens) [inline src/module/lighthouse/index.ts]: What about smashing `config` in here too? The benefit, `options` is now configurable. Downside, there may be extra options passed to lighthouse that will be disregarded and may cause collisions. Could always pick an array of keys off `config` to throw into `options` if you are worried abbout that. This way, a config file would be:

```yaml
configs:
  lighthouse:
    maxWaitForFcp: 120000
```

Comment (grgur) [reply src/module/lighthouse/index.ts]: Yes, that'd be fantastic. We'd need to make it universal for all modules.
Also, I'd prefer `maxWaitForFcp` to be larger than the lighthouse default as it seems like a common issue

Comment (mitchellsimoens) [reply src/module/lighthouse/index.ts]: What would be fantastic? Smashing config there blindly or having a way to pick props off the config? Here, you could:

```javascript
const options = {
  ...defaults,
  maxWaitForFcp: Config.get('configs.lighthouse.maxWaitForFcp'),
  ...userOptions,
};
```

or whatever order you'd want. If you want a better api to smash an array of keys, that can be done too.

Comment (grgur) [reply src/module/lighthouse/index.ts]: Well I would smash the defaults and load config from the user config file. There's already a backlog item for that effort, I believe.

Comment (mitchellsimoens) [reply src/module/lighthouse/index.ts]: I opened a PR into your branch to describe what I was thinking: #107

**Pull request merged**